### PR TITLE
[LOPS-2264] Remove binding usage for new_relic related scripts.

### PR DESCRIPTION
**Pull request by kporras07**

### CODE REVIEW

File: new_relic_apdex_t/new_relic_apdex_t.php [hunk 1]
```diff
@@ -30,27 +32,26 @@
...
+  $site_name = trim($req['body']);
+  $site_name = trim($site_name, '"');
+  $app_name = sprintf( "%s (%s)", $site_name, $env );
+  $output['app_name'] = $app_name;
 
-        return;
-      }
-      $output['api_key'] = $data['api_key'];
+  // Now get secrets for this site.
+  $req = pantheon_curl( 'https://customer-secrets.svc.pantheon.io/site/secrets' );
```
Comment (greg-1-anderson) [inline new_relic_apdex_t/new_relic_apdex_t.php:42]: Use `pantheon_get_secret()` instead.

File: new_relic_apdex_t/new_relic_apdex_t.php [hunk 2]
```diff
@@ -30,27 +32,26 @@
...
 function get_nr_connection_info( $env = 'dev' ) {
   $output = array();
-  $req    = pantheon_curl( 'https://api.live.getpantheon.com/sites/self/bindings?type=newrelic', null, 8443 );
-  $meta   = json_decode( $req['body'], true );
-
-  foreach ( $meta as $data ) {
-    if ( $data['environment'] === $env ) {
-      if ( empty( $data['api_key'] ) ) {
-        echo "Failed to get API Key\n";
+  $req    = pantheon_curl( 'https://api.live.getpantheon.com/sites/self/name', null, 8443 );
```
Comment (greg-1-anderson) [inline new_relic_apdex_t/new_relic_apdex_t.php:35]: We shouldn't use `pantheon_curl` in sample code. That mistake was first made back in 2016, but if the only thing we need is the site name, that's available in `$_ENV('PANTHEON_SITE_NAME')`.

**Review by greg-1-anderson — APPROVED** [review]

**Comment by kporras07** [comment]
@greg-1-anderson I updated the other 2 scripts; re-requested a review in case you want to take another look

File: new_relic_apdex_t/README.md [hunk 3]
```diff
@@ -17,9 +17,11 @@ To do the actual setting of default values this script first gets an API key and
 
 To use this example:
 
-1. [Activate New Relic Pro](https://pantheon.io/docs/new-relic/#activate-new-relic-pro) within your site dashboard. 
-2. Add the example `new_relic_apdex_t.php` script to the `private/scripts` directory of your code repository.
-3. Optionally, modify the environment to pull existing threshold T values from at the top of the file. This defaults to `dev` but can also be `test` or `live`.
+1. [Activate New Relic Pro](https://pantheon.io/docs/new-relic/#activate-new-relic-pro) within your site dashboard.
+2. Get a [New Relic User Key](https://docs.newrelic.com/docs/apis/intro-apis/new-relic-api-keys/)
+3. Using [Terminus Secrets Manager Plugin](https://github.com/pantheon-systems/terminus-secrets-manager-plugin), set a site secret for the API key just created (e.g. `new_relic_api_key`, if you name it something else, make sure to update in the script below). Make sure type is `runtime` and scope contains `web`.
```
Comment (namespacebrian) [inline new_relic_apdex_t/README.md:22]: Could add an example terminus command here

File: new_relic_deploy/README.md [hunk 4]
```diff
@@ -10,10 +10,12 @@ This script uses a couple clever tricks to get data about the platform. First of
...
 
-1. [Activate New Relic Pro](https://pantheon.io/docs/new-relic/#activate-new-relic-pro) within your site Dashboard. 
-2. Add the example `new_relic_deploy.php` script to the `private` directory of your code repository.
-3. Add a Quicksilver operation to your `pantheon.yml` to fire the script after a deploy.
-4. Test a deploy out!
+1. [Activate New Relic Pro](https://pantheon.io/docs/new-relic/#activate-new-relic-pro) within your site Dashboard.
+2. Get a [New Relic User Key](https://docs.newrelic.com/docs/apis/intro-apis/new-relic-api-keys/)
+3. Using [Terminus Secrets Manager Plugin](https://github.com/pantheon-systems/terminus-secrets-manager-plugin), set a site secret for the API key just created (e.g. `new_relic_api_key`, if you name it something else, make sure to update in the script below). Make sure type is `runtime` and scope contains `web`.
+4. Add the example `new_relic_deploy.php` script to the `private` directory of your code repository.
+5. Add a Quicksilver operation to your `pantheon.yml` to fire the script after a deploy.
```
Comment (namespacebrian) [inline new_relic_deploy/README.md:17]: I noticed that this step isn't mentioned in the `new_relic_apdex_t`, even though it's a needed step there as well.

This inconsistency was not introduced by this PR.  It was already inconsistent.

**Review by namespacebrian — APPROVED** [review]

**Pull request merged**